### ISSUE TITLE
Usage of texture format whitch is not supported by device

### ISSUE THREAD

**Issue by noahastor** [Try/OpenGothic#416]
Hello. 

Thank you very much for your hard work.

I am trying to run OpenGothic (with Gothic II: NotR) on an iMac running macOS Ventura 13.2.1 (with Pro 570 4gb graphics as listed below) compiled with HomeBrew however I am getting the following error: 

"OpenGothic v1.0 dev
no *.ini file in path - using default settings
no *.ini file in path - using default settings
GPU = AMD Radeon Pro 570
Depth format = Depth32F Shadow format = Depth16

---crashlog(std::system_error(Usage of texture format whitch is not supported by device))---
GPU: AMD Radeon Pro 570

zsh: abort      /Users/user/OpenGothic/build/opengothic/Gothic2Notr -g2 -g"

(The error appears without the "-g2" argument as well.)

Any input would be much appreciated.

Thank you.

**Comment by noahastor** [comment]
Hi @Try 

Thank you for the reply, much appreciated.

"Call-stack or maybe actual format name?"
How would I go about checking that information?

Thank you.

**Comment by Try** [comment]
> How would I go about checking that information?

I've had assumption that you build game from source, than it's trivial in XCode. Yet if you downloaded prebuilt version, than just wait for next update - I'll add extra logging

**Comment by Try** [comment]
Updated crash info, from Mac Source Ports :
```c++
OpenGothic v1.0 dev
no *.ini file in path - using default settings
no *.ini file in path - using default settings
unable to load archive: "/users/myusername/Gothic II/Data/.DS_Store", reason: VDF signature not recognized: "
GPU = Apple M1
Depth format = Depth32F Shadow format = Depth16

---crashlog(std::system_error(Depth32F: Usage of texture format whitch is not supported by device))---
GPU: Apple M1
```

**Comment by Try** [comment]
> iMac running macOS Ventura 13.2.1 (with Pro 570 4gb graphics as listed below)

Give this spec my best guess so far is that engine assumes that maximum texture size is 4k, but on your machine you need larger one, for depth buffer. Now engine will query max size that supported by GPU.

**Comment by Try** [comment]
should be fixed by [1f0bf6d](https://github.com/Try/OpenGothic/commit/1f0bf6d9ff1b262b5eb6b6c5e746e3ceae7ff0fa)
